lib/endpoints/discover.js

```diff
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright 2020 Joyent, Inc.
```

Comment:
This looks like now the only change is the copyright? In that case, it doesn't need to be modified.

Reply:
@bahamat I only changed the copyright year because `make check` complained about it.

Reply:
If you revert that does it still complain?

Reply:
Yes:
```
[root@build /data/triton-cmon]# make check
xmllint --noout --path deps/eng/tools/ --dtdvalid deps/eng/tools/service_bundle.dtd.1 smf/manifests/cmon.xml
/data/triton-cmon/build/node/bin/node node_modules/.bin/eslint  ./bin/cmon lib/errors.js lib/app.js lib/endpoints/_ping.js lib/endpoints/metrics.js lib/endpoints/discover.js lib/updater.js lib/common.js
deps/javascriptlint/build/install/jsl --nologo --nosummary --conf=tools/jsl.node.conf ./bin/cmon lib/errors.js lib/app.js lib/endpoints/_ping.js lib/endpoints/metrics.js lib/endpoints/discover.js lib/updater.js lib/common.js
/data/triton-cmon/bin/cmon
/data/triton-cmon/lib/errors.js
/data/triton-cmon/lib/app.js
/data/triton-cmon/lib/endpoints/_ping.js
/data/triton-cmon/lib/endpoints/metrics.js
/data/triton-cmon/lib/endpoints/discover.js
/data/triton-cmon/lib/updater.js
/data/triton-cmon/lib/common.js
deps/jsstyle/jsstyle -f tools/jsstyle.conf ./bin/cmon lib/errors.js lib/app.js lib/endpoints/_ping.js lib/endpoints/metrics.js lib/endpoints/discover.js lib/updater.js lib/common.js
deps/eng/tools/check-copyright -q
error: lib/endpoints/discover.js: copyright year not updated to 2022: ' * Copyright 2020 Joyent, Inc.'
make: *** [deps/eng/tools/mk/Makefile.targ:312: check-copyright] Error 1
[root@build /data/triton-cmon]# echo $?
2
```

```
[root@build /data/triton-cmon]# git diff
diff --git a/lib/endpoints/discover.js b/lib/endpoints/discover.js
index e2c6e9f..16a08ac 100644
--- a/lib/endpoints/discover.js
+++ b/lib/endpoints/discover.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright 2022 Joyent, Inc.
+ * Copyright 2020 Joyent, Inc.
  */
 
 // jsl:ignore
```